pkgs/sketch_pad/lib/editor/editor.dart

```diff
@@ -377,37 +398,37 @@ enum CompletionType {
 }
 
 extension CompletionSuggestionExtension on services.CompletionSuggestion {
-  HintResult toHintResult() {
-    var altDisplay = completion;
-    if (elementKind == 'FUNCTION' ||
-        elementKind == 'METHOD' ||
-        elementKind == 'CONSTRUCTOR') {
-      altDisplay = '$altDisplay()';
-    }
-
-    return HintResult(
-      completion,
-      displayText: displayText ?? altDisplay,
-      className: this.deprecated ? 'deprecated' : null,
-    );
-  }
+  // HintResult toHintResult() {
+  //   var altDisplay = completion;
+  //   if (elementKind == 'FUNCTION' ||
+  //       elementKind == 'METHOD' ||
+  //       elementKind == 'CONSTRUCTOR') {
+  //     altDisplay = '$altDisplay()';
+  //   }
+  //
+  //   return HintResult(
+  //     completion,
+  //     displayText: displayText ?? altDisplay,
+  //     className: this.deprecated ? 'deprecated' : null,
+  //   );
+  // }
 }
 
 extension SourceChangeExtension on services.SourceChange {
-  HintResult toHintResult() {
-    return HintResult(message, hintApplier: _applySourceChange);
-  }
-
-  void _applySourceChange(
-      CodeMirror editor, HintResult hint, Position? from, Position? to) {
-    final doc = editor.doc;
-
-    for (final edit in edits) {
-      doc.replaceRange(
-        edit.replacement,
-        doc.posFromIndex(edit.offset),
-        doc.posFromIndex(edit.offset + edit.length),
-      );
-    }
-  }
+  // HintResult toHintResult() {
```

Comment:
And this is used when we want to convert an analysis server quick fix or quick assist to a codemirror HintResult.